x-pack/plugins/observability/public/components/app/section/apm/index.test.tsx

```diff
@@ -105,8 +105,8 @@ describe('APMSection', () => {
       <APMSection bucketSize={{ intervalString: '60s', bucketSize: 60 }} />
     );
 
-    expect(getByText('APM')).toBeInTheDocument();
-    expect(getByText('View in app')).toBeInTheDocument();
+    expect(getByText('Services')).toBeInTheDocument();
+    expect(getByText('Show service inventory')).toBeInTheDocument();
     expect(getByText('Services 11')).toBeInTheDocument();
```

Comment:
@estermv Sorry for pulling you in here, but I don't know enough about Jest works 🙈 It seems that the test is complaining that I basically have the same check for `Services` and I need to use some other form of check for text. It proposes `queryAllByText` but that doesn't seem to be supported. Any ideas on how this should be changed to fit with the new panel title? Thanks 👍

Reply:
@formgeist I just push a commit that should fix the tests. The problem was that now the text `Services` appears twice in the component and `getByText` fails when there is more than one occurrence of the text you pass as a parameter.

I replaced it to use `getByRole` and query for a heading element. These queries are provided by the [testing-library](https://testing-library.com/docs/), which is not part of Jest.

I also fix a test that was just replacing the text.

Reply:
Thanks @estermv 🙌

Reply:
@formgeist I pushed also a commit fixing the checks

Reply:
Thanks, appreciate the help with this 👍